auditlog_security/models/auditlog_log_line.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022-2024 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.osv.expression import OR
+
+
+class AuditlogLogLine(models.Model):
+    _inherit = "auditlog.log.line"
+    _order = "create_date desc"
+
+    user_id = fields.Many2one(related="log_id.user_id")
+    method = fields.Char(related="log_id.method")
+    model_id = fields.Many2one(related="log_id.model_id", store=True)
```

Comment:
can we get rid of storing this?

Reply:
Actually we need it stored for filtering purposes in views etc

Reply:
... on the other hand, the storing of these fields is a bit of a downer if you have auditlog installed, a lot of lines in the system, and now you want to install this module but it takes ages for the stored fields to populate.

@ntsirintanis non-stored related fields also support searching and grouping, right? Maybe we can live without the sort option, and then remove the storedness of these fields.

Reply:
`create index auditlog_log_model_id_res_id on auditlog_log(model_id, res_id);`

Reply:
auto_join on log_id should help here